btrack/dataio.py

```diff
@@ -570,7 +570,7 @@ def write_objects(self, data):
         self.write_properties(props)
 
     @h5check_property_exists('objects')
-    def write_properties(self, data: dict):
+    def write_properties(self, data: dict, overwrite: bool = False):
```

Comment:
```suggestion
    def write_properties(self, data: dict, allow_overwrite: bool = False):
```
We should make this parameter a bit more meaningful

Reply:
Can you also add it to the docstrings below